### PR TITLE
Add library DynamicODataToSQL

### PR DESCRIPTION
**Pull request by vaibhav-goyal**
Add library DynamicODataToSQL

### CODE REVIEW

**Comment by vaibhav-goyal** [comment]
Hi @marabooy requesting your review to publish this library in Odata libraries for dotnet

**Review by marabooy — APPROVED** [review]

**Comment by marabooy** [comment]
@vaibhav-goyal thank you for the pr. Having the team review :)

File: _libraries/DynamicODataToSQL.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+category: net
+name: DynamicODataToSQL
+link: https://github.com/DynamicODataToSQL/DynamicODataToSQL
+version: V4
+object: Server
+downloads:
+  - source: nugetgallery
+    link: https://www.nuget.org/packages/DynamicODataToSQL/
+---
+Library to convert OData query directly to SQL query without IQuearable or entity framework. It is useful when the data model is dynamic and hence entity framework or any other ORM with IQuerable support cannot be used.
```
Comment (KenitoInc) [inline _libraries/DynamicODataToSQL.md:11]:
```suggestion
Library to convert OData query directly to SQL query without IQueryable or entity framework. It is useful when the data model is dynamic and hence entity framework or any other ORM with IQueryable support cannot be used.
```

**Review by KenitoInc — CHANGES_REQUESTED** [review]

**Comment by vaibhav-goyal** [comment]
Thank you @marabooy and @KenitoInc for review.

**Review by gathogojr — APPROVED** [review]

**Review by KenitoInc — APPROVED** [review]

**Pull request merged**